Specs/DateTime/French/DateTimeModel.json

```diff
@@ -3202,210 +3202,17153 @@
     ]
   },
   {
-    "Input": "Je retournerai 4:45 du matin",
```

Comment:
No previously existing case should be removed. Unless it is clearly incorrect and this should be noted in the PR description.
If a duplicate case was inserted, please remove the new one, not the original. This faciliates merging as multiple people may be working over the same files.